### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/rsend/send-cD.ksh [hunk 1]
```diff
@@ -45,7 +45,7 @@ typeset inc=$BACKDIR/stream.inc
 log_must zfs create -o compress=lz4 $sendfs
 log_must zfs create -o compress=lz4 $recvfs
 typeset dir=$(get_prop mountpoint $sendfs)
-# Don't use write_compressible: we want compressible but undedupable data here.
+# Don't use write_compressible: we want compressible but undeduplicable data here.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/rsend/send-cD.ksh]: Tell me if you like the new version.

File: tests/zfs-tests/tests/functional/rsend/send-cD.ksh [hunk 2]
```diff
@@ -45,7 +45,7 @@ typeset inc=$BACKDIR/stream.inc
 log_must zfs create -o compress=lz4 $sendfs
 log_must zfs create -o compress=lz4 $recvfs
 typeset dir=$(get_prop mountpoint $sendfs)
-# Don't use write_compressible: we want compressible but undeduplicable data here.
+# Don't use write_compressible: we want compressible but no-dedup data here.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/rsend/send-cD.ksh]: Ok! I squash!

File: tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh [hunk 3]
```diff
@@ -38,7 +38,7 @@
 # STRATEGY:
 #	1. Separately promote pool clone, filesystem clone and volume clone.
 #	2. Recursively backup all the POOL and restore in POOL2
-#	3. Verify all the datesets and property be properly received.
+#	3. Verify all the datasets and property be properly received.
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh]:
```suggestion
#	3. Verify all the datasets and properties are properly received.
```

File: tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh [hunk 4]
```diff
@@ -39,7 +39,7 @@
 #	1. Setting properties for all the filesystem and volumes randomly
 #	2. Backup all the data from POOL by send -R
 #	3. Restore all the data in POOL2
-#	4. Verify all the perperties in two pools are same
+#	4. Verify all the properties in two pools are same
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh]:
```suggestion
#	4. Verify all the properties in two pools are the same
```

File: tests/zfs-tests/tests/functional/rsend/send-cD.ksh (hunk 1 above)
[the remark this answers is not in the document]
Comment (freqlabs) [reply tests/zfs-tests/tests/functional/rsend/send-cD.ksh]: :+1:

File: tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh (hunk 4 above)
Comment (rlaager) [reply tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh]: "in the two pools"

File: tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh (hunk 3 above)
Comment (Gelma) [reply tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh]: Done!

File: tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh (hunk 4 above)
Comment (Gelma) [reply tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh]: Done!

File: tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh (hunk 3 above)
Comment (freqlabs) [reply tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh]: It still says "be" not "are" (but please make it "were" instead).

Comment (Gelma) [reply tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh]: Done!

**Pull request merged**